_i18n/en.yml

```diff
@@ -523,11 +523,11 @@ help_pages:
       For additional help, [please submit a support ticket] (https://login.gov/contact).
```

Comment:
Just noticing this line seems to have broken markdown syntax (extra space), can we fix it while we're here?

Reply:
Sure